libeos-payg/efi.c

```diff
@@ -527,7 +527,7 @@ eospayg_efi_securebootoption_disabled (void)
  * space, but the kernel has a placeholder file for it due to a failed
  * write.
  *
- * Returns: size of the PK efi variable including EFI overhead, or -1 if missing
+ * Returns: size of the PK EFI variable, excluding the 4-byte attribute overhead, or -1 if missing
```

Comment:
Do we have 80 characters rule here?  Others are LGTM!
If it is okay, feel free to merge it.

Reply:
It's not documented but you're right: most of the rest of the file, and the rest of this docstring, has a column limit. I amended this commit to wrap this line. Thansk